clearwater-config-manager.root/usr/share/clearwater/clearwater-config-manager/scripts/apply_shared_config

```diff
@@ -32,7 +32,17 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
-FILENAME=/etc/clearwater/shared_config.apply
+[ $# -le 1 ] || { echo "Usage: apply_shared_config [--sync]" >&2 ; exit 2 ; }
```

Comment:
Can we check that `$1` is `--sync`? I'm nervous that someone will run `apply_shared_config -h` to try and get a usage statement.